code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -314,3 +430,511 @@
 	pyjamas["blue pyjamas"] = /obj/item/clothing/under/pj/blue
 	pyjamas["red pyjamas"] = /obj/item/clothing/under/pj/red
 	gear_tweaks += new/datum/gear_tweak/path(pyjamas)
+
+/datum/gear/uniform/sari
+	display_name = "sari (selection)"
+	path = /obj/item/clothing/under/dress/sari
+
+/datum/gear/uniform/sari/New()
+	..()
+	var/sari = list()
+	sari["red sari"] = /obj/item/clothing/under/dress/sari
+	sari["green sari"] = /obj/item/clothing/under/dress/sari/green
+	gear_tweaks += new/datum/gear_tweak/path(sari)
+
+/datum/gear/uniform/wrappedcoat
+	display_name = "modern wrapped coat"
+	path = /obj/item/clothing/under/moderncoat
+
+/datum/gear/uniform/ascetic
+	display_name = "plain ascetic garb"
+	path = /obj/item/clothing/under/ascetic
+
+/datum/gear/uniform/overalls_outfit
+	display_name = "overalls (selection)"
+	path = /obj/item/clothing/under/overalls
+
+/datum/gear/uniform/overalls_outfit/New()
+	..()
+	var/overalls_outfits = list()
+	overalls_outfits["overalls"] = /obj/item/clothing/under/overalls
+	overalls_outfits["sleek overalls"] = /obj/item/clothing/under/overalls/sleek
+	overalls_outfits["service overalls"] = /obj/item/clothing/under/serviceoveralls
+	gear_tweaks += new/datum/gear_tweak/path(overalls_outfits)
+
+/datum/gear/uniform/utility
+	display_name = "utility jumpsuit (selection)"
```

Comment:
If a bit confused: these are jumpsuits which I would've said bears a resemblance towards overalls used in fields that involve heavy-duty work, including (besides the mentioned in the utility pants reply) mechatronic engineering, construction, and even R&D.
If not a bit confused and determined on them being military uniforms: see above; and, utility jumpsuits aren't necessarily military. If it's by the consequence that the military happens to adopt such wear because of their practicality, I don't really see how that suddenly makes its 'civilian' counterpart something that is prohibited. There are no chevrons, no stripes, no stars, no medals on these loadout uniforms. Please understand that I didn't just "dump" them in for no apparent reason. We already have TCFL clothes and military fashion even if these inherently and innately denoted a military association.